charts/rollups-node/script.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
set -euo pipefail
```

For better portability